pkgs/top-level/perl-packages.nix

```diff
@@ -10973,11 +10974,21 @@ let
         # add runtime dependencies that cause silent failures when missing
         wrapProgram $out/bin/$file --prefix PATH : ${lib.makeBinPath [ pkgs.ghostscript pkgs.potrace ]}
       done
+      # do NOT pass TEXMF=$tex in makeMakerFlags, or the build will (try to) launch mktexlsr
+      # instead, install the TeX styles manually
+      install -d -m 755 $tex/tex/latex/latexml
+      install -m 644 blib/lib/LaTeXML/texmf/*.sty $tex/tex/latex/latexml
     '';
     meta = {
       description = "Transforms TeX and LaTeX into XML/HTML/MathML";
       license = lib.licenses.free;
     };
+  }) {
+    tlType = "bin";
+    pkgs = [ LaTeXML LaTeXML.tex ] ++
+      (builtins.concatMap (builtins.getAttr "pkgs") (with pkgs.texlive; [ dvipng dvisvgm ]));
```

Comment:
```suggestion
      (with pkgs.texlive; dvipng.pkgs ++ dvisvgm.pkgs);
```

Reply:
Also, shouldn't it be something like
```
deps = [ "dvipng" "dvisvgm" ];
```

Reply:
`texlive.combine` calls [`combinePkgs`](https://github.com/NixOS/nixpkgs/blob/cf9bf19bf32f8866b19fca02705e02be204a601d/pkgs/tools/typesetting/tex/texlive/default.nix#L163), which expects `pkgs` to have already been resolved into the reflexive transitive closure of `deps`, which motivates the contorsions with the self-dependencies.

Reply:
I dropped the dependencies https://github.com/NixOS/nixpkgs/pull/118743#discussion_r625233883 so now it's plain `pkgs = [ LaTeXML.tex ]`.